example-notebooks/Load_test_terra_set_up.ipynb

```diff
@@ -0,0 +1,289 @@
+    "    api_client.configuration.host = wds_url\n",
+    "    \n",
+    "    # records client is used to interact with Records in the data table\n",
+    "    records_client = wds_client.RecordsApi(api_client)\n",
+    "    # data to upload to wds table\n",
+    "    \n",
+    "    # Upload entity to workspace data table with name \"testType_uploaded\"\n",
+    "    response = records_client.upload_tsv(current_workspaceId, version, recordName, tsv_file_name)\n",
+    "    print(response)\n",
+    "\n",
+    "# KICK OFF A WORKFLOW INSIDE A WORKSPACE\n",
+    "def submit_workflow_assemble_refbased(workspaceId, dataFile):\n",
+    "    cbas_url = get_app_url(workspaceId, \"cbas\")\n",
+    "    print(cbas_url)\n",
+    "    #open text file in read mode\n",
+    "    text_file = open(dataFile, \"r\")\n",
+    "    request_body2 = text_file.read();\n",
+    "    text_file.close()\n",
+    "    \n",
+    "    uri = f\"{cbas_url}/api/batch/v1/run_sets\"\n",
+    "    \n",
+    "    headers = {\"Authorization\": azure_token,\n",
+    "               \"accept\": \"application/json\", \n",
+    "              \"Content-Type\": \"application/json\"}\n",
+    "    \n",
+    "    response = requests.post(uri, data=request_body2, headers=headers)\n",
+    "    # example of what it returns: {'run_set_id': 'cdcdc570-f6f3-4425-9404-4d70cd74ce2a', 'runs': [{'run_id': '0a72f308-4931-436b-bbfe-55856f7c1a39', 'state': 'UNKNOWN', 'errors': 'null'}, {'run_id': 'eb400221-efd7-4e1a-90c9-952f32a10b60', 'state': 'UNKNOWN', 'errors': 'null'}], 'state': 'RUNNING'}\n",
+    "    print(response.json())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5f54cbc2-9483-490e-be84-26b8ddc62640",
+   "metadata": {},
+   "source": [
+    "# This section executes the load test. \n",
+    "\n",
+    "Unless you want to change the behavior of the test, no need to make any changes to this section - output is mostly for debugging reason to make sure things are moving along. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "47d15199-2813-4685-a125-07bec591f1fc",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# get existing workspaces\n",
+    "api_call1 = f\"{workspace_manager_url}/api/workspaces/v1\";\n",
+    "header = {\"Authorization\": azure_token};\n",
+    "\n",
+    "#response = requests.get(url=api_call1, headers=header)\n",
+    "#print(response.json())\n",
+    "\n",
+    "workspaceIds = []\n",
+    "\n",
+    "while number_of_workspaces != 0:\n",
+    "    # create the specified number os workspaces\n",
+    "    workspaceId = create_workspace(cbas)\n",
+    "    workspaceIds.append(workspaceId)\n",
+    "    number_of_workspaces-=1\n",
+    "\n",
+    "\n",
+    "# track to see when the workspace WDS are ready to upload data into them \n",
+    "# sleep for 5 minutes to allow WDS to start up, if no wds, only sleep 2 minutes to let cbas start up\n",
+    "if wds_upload:\n",
+    "    time.sleep(300)\n",

```

Comment:
Sadly I think we need to sleep for more than 5 minutes - I tested this out and it timed out, but the WDS did get successfully created